cmd/podman/containers/exec.go

```diff
@@ -110,6 +111,12 @@ func exec(_ *cobra.Command, args []string) error {
 
 	execOpts.Envs = envLib.Join(execOpts.Envs, cliEnv)
 
+	for fd := 3; fd < int(3+execOpts.PreserveFDs); fd++ {
+		if !rootless.IsFdInherited(fd) {
+			return errors.Errorf("failed to verify %d file descriptor", fd)
```

Comment:
This is confusing. Shouldn't it read `file descriptor %d`?

Reply:
Actually" failed to verify" itself doesn't really make sense to me as an end user. How about "file descriptor %d is not open" or "...is not available for I/O" or something along those lines? @giuseppe since you have the best understanding of what this code is actually validating, could you suggest a phrasing that is both technically accurate and helpful to the end user?

Reply:
The code is verifying it a file descriptor actually exists.  I think "file descriptor %d is not open"  is correct.
Users are telling podman to leak file descriptor 3 and 4 in to a container, and if 3 and 4 are not open, then something went wrong.

Reply:
Maybe `"file descriptor %d is not available - the preserve-fds option requires that file descriptors must be passed"`